File: solutions/349.intersection-of-two-arrays/intersection-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[]}
 */
const intersection = (nums1, nums2) =>{
  let set = new Set(nums1)
  let output= new Set()
  nums2.forEach(e=> {
    if (set.has(e)) output.add(e)
  })
  return [...output]
}

File: solutions/349.intersection-of-two-arrays/intersection-of-two-arrays.py
class Solution:
    def intersection(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[int]
        """
        s = set(nums1)

        ret = set()
        for n in nums2:
            if n in s:
                ret.add(n)

        return list(ret)
